dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtilTest.java

```diff
@@ -66,8 +67,8 @@ public void testCreateFork() throws Exception {
         verify(repo, times(1)).fork();
     }
 
-    @Test
-    public void testCreateForkWithExceptionProceeds() throws Exception {
+    @Test(expectedExceptions = UnableToForkException.class)
```

Comment:
Probably should be it's own new test since other exceptions should probably proceed.

Reply:
Other exceptions can also cause indefinite loop. So for any exception thrown during forking we should propagate it up.

Reply:
Actually it should not proceed if it fails to create a fork. Otherwise we would run into indefinite loop again.